src/pages/topics/[slug].tsx

```diff
@@ -49,32 +57,82 @@ const Topic: NextPage<TopicProps> = ({
                 description={description}
                 ctas={ctas}
             />
-            <div sx={{ padding: ['inc40', null, 'inc50', 'inc70'] }}>
-                <GridLayout sx={{ rowGap: ['inc90', null, 'inc130'] }}>
-                    <TopicCardsContainer topics={topics} name={name} />
-                    <FeaturedCardSection content={featured} />
-                    {tutorials.length > 2 && (
-                        <CardSection content={tutorials} title="Tutorials" />
-                    )}
-                    {demoApps.length > 2 && (
-                        <CardSection content={demoApps} title="Demo Apps" />
-                    )}
-                    {articles.length > 2 && (
-                        <CardSection content={articles} title="Articles" />
-                    )}
-                    {videos.length > 2 && (
-                        <CardSection content={videos} title="Videos" />
-                    )}
-                    {podcasts.length > 2 && (
-                        <CardSection
-                            content={podcasts}
-                            title="Podcasts"
-                            direction="column"
-                        />
-                    )}
-                </GridLayout>
+            <div
+                sx={{
+                    display: 'flex',
+                    flexDirection: 'column',
+                    paddingBottom: 'inc160',
+                    px: ['inc40', null, 'inc50', 'inc70'],
+                    paddingTop: ['inc40', null, 'inc50', 'inc70'],
+
+                    rowGap: ['inc90', null, null, 'inc130'],
+                }}
+            >
+                {(variant !== 'light' || topics.length > 0) && (
+                    <div>
+                        <GridLayout
+                            sx={{ rowGap: ['inc90', null, null, 'inc130'] }}
+                        >
+                            {topics.length > 0 && (
+                                <TopicCardsContainer
+                                    topics={topics}
+                                    title={`${name} Topics`}
+                                />
+                            )}
+                            {variant !== 'light' && (
+                                <FeaturedCardSection content={featured} />
+                            )}
+                            {variant === 'heavy' && (
+                                <>
+                                    {tutorials.length > 2 && (
```

Comment:
Let me know what you think about making this into a map so that we don't have to repeat the same code. If you think its an overkill I am ok with leaving it as is. 

```
{[tutorials, demoApps, articles, videos, podcasts,].map((section,i) => {
    if (section.length > 2) {
        return (
            <CardSection
                key={section.category}
                content={section}
                title={section.category}
            />
        )
    }
}
```

Reply:
Ah so i actually converted it into a map in [this PR](https://github.com/mongodb/devcenter/pull/39). I'll merge main into this branch and that should resolve it.